components/CardItem.vue

```diff
@@ -100,13 +100,13 @@ export default {
     padding-bottom: $offset-1;
 
     @media (max-width: $query-mobile) {
-      font-size: $f-3;
-      line-height: $f-3;
+      font-size: $f-4;
+      line-height: $f-4;
     }
 
     @media (min-width: $query-mobile) and (max-width: $query-mobile-landscape) and (orientation: landscape) {
-      font-size: $f-4;
-      line-height: $f-4;
+      font-size: calc($f-5 + 0.1rem);
+      line-height: calc($f-5 + 0.1rem);
```

Comment:
too bad we need such a hacky solution while maintaining our own custom type scale